### PR TITLE
Leaderboard

### PR DESCRIPTION
**Pull request by ColDog**
closes battlesnakeio/roadmap#18.

### CODE REVIEW

File: play/apps/leaderboard/models.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    user_snake = models.ForeignKey(
+        UserSnake, primary_key=True, on_delete=models.CASCADE
+    )
+
+    @classmethod
+    def ranked(cls):
+        snakes = list(UserSnakeLeaderboard.objects.all())
+        return sorted(snakes, key=lambda s: s.rank())
+
+    def rank(self):
```
Comment (ColDog) [inline play/apps/leaderboard/models.py]: Need to make this more interesting.

**Comment by dlsteuer** [comment]
@ColDog is there a reason for the styling changes? imo those decisions should come down from the repo code owners

**Comment by codeallthethingz** [comment]
Colin uses Black which is the python autoformatter.

**Comment by codeallthethingz** [comment]
@ColDog remember.

**Pull request merged**